src/asmcnc/production/lower_beam_qc_jig/lower_beam_qc.py

```diff
@@ -330,8 +331,7 @@ def enable_alarms(self):
 		self.m.s.write_command('$21 = 1')
 
 	def shutdown_console(self):
```

Comment:
Why have this function in the first place? just call console_utils.shutdown() directly